modules/cluster-logging-collector-log-forward-gcp.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+// cluster-logging-external.adoc
+//
+
+:_content-type: PROCEDURE
+[id="cluster-logging-collector-log-forward-gcp_{context}"]
+= Forwarding logs to Google Cloud Platform (GCP)
+
+You can forward logs to link:https://cloud.google.com/logging/docs/basic-concepts[Google Cloud Logging] in addition to, or instead of, the internal default {product-title} log store.
+
+[NOTE]
+====
+Using this feature with Fluentd is not supported.
+====
+
+.Prerequisites
+* {logging-title-uc} Operator 5.5.1 and later
+
+
+.Procedure
+
+. Create a `ClusterLogForwarder` Custom Resource YAML using the template below:
+
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1"
+kind: "ClusterLogForwarder"
+metadata:
+  name: "instance"
+  namespace: "openshift-logging"
+spec:
+  outputs:
+    - name: gcp-1
+      type: googleCloudLogging
+      secret:
+        name: gcp-secret
+      googleCloudLogging: <1>
+        projectId : "openshift-gce-devel"
+        logId : "app-gcp" <2>
+  pipelines:
+    - name: test-app
+      inputRefs:
```

Comment:
We can add the <3> point here for inputRefs with description. _Specify which log types to forward by using the pipeline: application, infrastructure, or audit._ Also remove the #- infrastructure